internal/query/projection/limits_test.go

```diff
@@ -21,7 +21,42 @@ func TestLimitsProjection_reduces(t *testing.T) {
 		want   wantReduce
 	}{
 		{
-			name: "reduceLimitsSet",
+			name: "reduceLimitsSet all properties",
```

Comment:
Please add tests for single properties reset

Reply:
I thought I did so, but will check again

Reply:
Yeah, there is the test called "reduceLimitsReset single property" (renamed it a bit now) and "reduceLimitsSet one property"

Reply:
Sorry, my bad, as it was already existing, but I would still test for all possible settings if you want to have the option to reset them all.

Reply:
obsolete as we don't have a reset event anymore